src/net/sourceforge/kolmafia/KoLCharacter.java

```diff
@@ -5049,28 +5051,27 @@ public static final Modifiers recalculateAdjustments(
     // Consider fake hands
     int fakeHands = EquipmentManager.getFakeHands();
     if (fakeHands > 0) {
-      newModifiers.add(
-          Modifiers.WEAPON_DAMAGE, -1 * fakeHands, "Hands:fake hand (" + fakeHands + ")");
+      newModifiers.add(Modifiers.WEAPON_DAMAGE, -1 * fakeHands, ModifierType.EQUIPMENT_POWER);
     }
 
     int brimstoneMonsterLevel = 1 << newModifiers.getBitmap(Modifiers.BRIMSTONE);
     // Brimstone was believed to affect monster level only if more than
     // one is worn, but this is confirmed to not be true now.
     // Also affects item/meat drop, but only one is needed
     if (brimstoneMonsterLevel > 1) {
-      newModifiers.add(Modifiers.MONSTER_LEVEL, brimstoneMonsterLevel, "Outfit:brimstone");
-      newModifiers.add(Modifiers.MEATDROP, brimstoneMonsterLevel, "Outfit:brimstone");
-      newModifiers.add(Modifiers.ITEMDROP, brimstoneMonsterLevel, "Outfit:brimstone");
+      newModifiers.add(Modifiers.MONSTER_LEVEL, brimstoneMonsterLevel, ModifierType.OUTFIT);
```

Comment:
I'm really not a fan of discarding the information in Outfit:brimstone -> Outfit.

If you have one piece of Cloathing and one piece of brimstone equipped, modtrace will print out the resulting item drop for both in a way you can differentiate between the two. Not so in this case.

Reply:
Fixed.